### DISCUSSION TITLE
hostname and hostname-short

### DISCUSSION THREAD

**Discussion by taflaj** [dracula/dracula-theme#910] [Ideas] [answered]
In my work environment, I have tmux running on multiple instances, such as servers, VMs, containers, etc. I'm using `set -g @dracula-show-left-icon hostname`, along with custom colors, so I can have a quick visual on which instance I'm currently on. The problem is that the full hostname, at times, can be extra long, thus taking valuable space. To solve this, I've added a new option, namely `shortname`, that only displays the short hostname (up until the first `.`).

I've created a pull request for this, in hopes it's approved.

Otherwise, it's an honor to be part of this community.

**Comment by luxonauta** [comment, marked as answer]
https://github.com/dracula/tmux/pull/240 Merged already.
